### PR TITLE
Richih/update 2019

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

**Comment by RichiH** [comment]
I don't see a merge locally, I merely cherry-picked from master locally, but nm, result should be OK.

**Comment by RichiH** [comment]
@juliusv

**Comment by brian-brazil** [comment]
:+1:

File: content/2019-munich/sponsor.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: Become a Sponsor
+---
+
+## Become a sponsor
+
+As a community conference with a low entrance fee, we depend on sponsors for
+making PromCon possible.
+
+Do you want to help make this happen and **get your company featured in videos,
+banners, and printed materials?
```
Comment (juliusv) [inline content/2019-munich/sponsor.md]: Missing closing `**` to end the boldness?

**Comment by juliusv** [comment]
Gonna merge and fix the missing `**` myself

**Pull request merged**